test/Microsoft.ML.Tests/TrainerEstimators/LbfgsTests.cs

```diff
@@ -80,14 +81,24 @@ public void TestLogisticRegressionStats_MKL()
                 s.StdComputer = new ComputeLRTrainingStdThroughHal();
             }));
 
-            var transformerChain = pipe.Fit(dataView) as TransformerChain<BinaryPredictionTransformer<ParameterMixingCalibratedPredictor>>;
+            var transformer = pipe.Fit(dataView) as TransformerChain<BinaryPredictionTransformer<ParameterMixingCalibratedPredictor>>;
 
-            var linearModel = transformerChain.LastTransformer.Model.SubPredictor as LinearBinaryModelParameters;
+            var linearModel = transformer.LastTransformer.Model.SubPredictor as LinearBinaryModelParameters;
             var stats = linearModel.Statistics;
             LinearModelStatistics.TryGetBiasStatistics(stats, 2, out float stdError, out float zScore, out float pValue);
 
             CompareNumbersWithTolerance(stdError, 0.250672936);
             CompareNumbersWithTolerance(zScore, 7.97852373);
+
+            var scoredData = transformer.Transform(dataView);
+
+            var coeffcients  = stats.GetCoefficientStatistics(linearModel, scoredData.Schema["Features"], 100);
+
+            Assert.Equal(19, coeffcients.Length);
```

Comment:
>coeffcients [](start = 29, length = 11)

Not a big deal at all since it's just a test, but while you're at it, this is a typo. `coeffcients` => `coefficients`.